config/config.exs

```diff
@@ -13,6 +13,7 @@ config :skate,
   # Default. Can be configured via environment variable, which is loaded in application.ex
   api_url: {:system, "API_URL"},
   api_key: {:system, "API_KEY"},
+  restrict_environment_access?: System.get_env("RESTRICT_ENVIRONMENT_ACCESS") == "true",
```

Comment:
**issue:** sorry I don't have a good explainer handy to link (I'm trying and failing to find the right spot in the docs to point to), but since the configuration is read in at compile time rather than runtime this won't reflect changes to this environment variable in different application environments. You'll want to use `config/releases.exs` (which is evaluated at runtime) instead.

Reply:
Ah yes, I remember [this doc](https://elixir-lang.org/getting-started/mix-otp/config-and-releases.html#configuration) about that. Thanks for catching!

Reply:
Though looking again that doc specifies the file `config/runtime.exs`, which seems like an evolution of `releases.exs`. Looks like `releases.exs` maybe isn't automatically used when running locally? But it does seem like the appropriate place for this configuration.

Reply:
From [the release notes for Elixir 1.11](https://elixir-lang.org/blog/2020/10/06/elixir-v1-11-0-released/), it looks like `config/releases.exs` works for us because it's specifically run at runtime in the context of a release, but doesn't do anything when (say) run locally via Mix. It looks like their recommended course of action is to just rename the file if you're already using `releases.exs` (though keeping it as-is sounds like it's still supported).

Reply:
(Though thanks for saying something, that led me to actually look up what the difference was - it has confused me before.)

Reply:
Thanks for pointing to that doc! I made the switch to `runtime` in [7b47e69](https://github.com/mbta/skate/pull/2030/commits/7b47e69ca23d9eca5e7669a74097fae5ada2c7cd) since it made this easier to test locally. I'll make a separate tech debt ticket for moving the [custom runtime configuration ](https://github.com/mbta/skate/blob/7053aed97e9281294b9f17c275bac7d485d5d3a8/lib/skate/application.ex#L12) to use `config/runtime.exs` insead